Exercises-1-Basic/src/exercises/Num1.java

```diff
@@ -0,0 +1,17 @@
+package exercises;
+
+/*
+ * Write a program to print numbers from 1 to 10.
+ */
+public class Num1 {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
```

Comment:
Remove this line, as it doesn't add any additional information.

Reply:
Done.